src/main/java/Todo.java

```diff
@@ -0,0 +1,12 @@
+public class Todo extends Task {
+    protected static final String TODO_ICON = "T";
+
+    public Todo(String taskName) {
+        super(taskName);
+    }
+
+    @Override
+    public String toString() {
+        return OPEN_SQUARE_BRACKET + TODO_ICON + CLOSE_SQUARE_BRACKET + super.toString();
+    }
```

Comment:
Similar to the earlier `Task.java` file, you can consider not abstracting out the open brackets and "T" into constants

Reply:
Neat and readable, well done.